src/pybind/mgr/dashboard/frontend/cypress.json

```diff
@@ -21,6 +21,9 @@
   "retries": 1,
   "env": {
     "LOGIN_USER": "admin",
-    "LOGIN_PWD": "admin"
-  }
+    "LOGIN_PWD": "admin",
+    "CEPH2_URL": "http://localhost:4202/"
```

Comment:
Probably https? As we have ssl enabled by default right?

Reply:
it doesn't matter actually. this will act as a template only. the URL for it is modified internally in the `run-frontend-e2e-tests.sh` file.